_docs/advanced-configurations/programming-escs-with-passthrough.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Programming ESCs with Passthrough"
+excerpt: "How to use BLHeliConfigurator or BLHeliSuite with dRonin"
+---
+
+{% include callout type="danger" title="Remove props before ESC programming!" text="ESC programming generally requires battery power to be provided to the ESCs, and for unusual signals to be sent down the control line to the ESC.  Motors can accidentally beign spinning.  Ensure that **ALL PROPS** are removed from the vehicle prior to ESC programming and that wires and fingertips will not become entangled in motors if they begin to spin." %}
+
+dRonin now supports programming ESCs using 4WIF-style passthrough.  The "MSP" module emulates the MultiWii/CleanFlight/BetaFlight configuration interface.  BLHeli, BLHeli_S , BLHeli32, and SimonK ESCs can be reprogrammed.
```

Comment:
Heh, not sure that sentence is strictly true but the point is made. :stuck_out_tongue: